### PR TITLE
2024-08-07 Release

### PR DESCRIPTION
**Pull request by anitacaron**

### CODE REVIEW

**Comment by anitacaron** [comment]
I'm checking why `uberon-base.owl` hasn't been updated.

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
Something did not work, since ontologies are identical

**Comment by Caroline-99** [comment]
I have checked it. Everything looks good to me.

File: src/ontology/diffs/uberon-diff.md [hunk 1]
```diff
@@ -1 +1,4030 @@
...
+#### Removed
+- [cortex of kidney](http://purl.obolibrary.org/obo/UBERON_0001225) [depiction](http://xmlns.com/foaf/0.1/depiction) "https://ccf-ontology.hubmapconsortium.org/objects/v1.2/VH_F_Kidney_R.glb"^^[anyURI](http://www.w3.org/2001/XMLSchema#anyURI) 
+
+- [cortex of kidney](http://purl.obolibrary.org/obo/UBERON_0001225) [depiction](http://xmlns.com/foaf/0.1/depiction) "https://ccf-ontology.hubmapconsortium.org/objects/v1.2/VH_M_Kidney_L.glb"^^[anyURI](http://www.w3.org/2001/XMLSchema#anyURI) 
+
+- [cortex of kidney](http://purl.obolibrary.org/obo/UBERON_0001225) [depiction](http://xmlns.com/foaf/0.1/depiction) "https://ccf-ontology.hubmapconsortium.org/objects/v1.2/VH_F_Kidney_L.glb"^^[anyURI](http://www.w3.org/2001/XMLSchema#anyURI) 
+
+- [cortex of kidney](http://purl.obolibrary.org/obo/UBERON_0001225) [depiction](http://xmlns.com/foaf/0.1/depiction) "https://ccf-ontology.hubmapconsortium.org/objects/v1.2/VH_M_Kidney_R.glb"^^[anyURI](http://www.w3.org/2001/XMLSchema#anyURI) 
+
+#### Added
```
Comment (aleixpuigb) [inline src/ontology/diffs/uberon-diff.md:962]: Does it mean that will display 4 objects? I think they don't want all of them (we should ask if there's a preffered gender and side?)

Comment (anitacaron) [reply src/ontology/diffs/uberon-diff.md:962]: I understood that they wanted only the latest version but keep the gender and side.

**Review by aleixpuigb — APPROVED** [review]
It looks good, we need to ask HuBMAP if they want all the object (male/female + R/L)

**Pull request merged**